### PR TITLE
Feat(ExplicitCancel): return false to retry explicitly

### PR DESCRIPTION
**Pull request by EdenTurgeman**
Failed and no return value will stop as long as the promise is resolved, return false to cancel or throw an exception.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,10 +23,16 @@
     "test:coverage": "cross-env jest --coverage && cat ./coverage/lcov.info | coveralls",
     "release": "npm run lint && npm run test:coverage && npm run build"
   },
+  "contributors": [{
+    "name" : "Eden Turgeman",
+    "email" : "edmx0.et@gmail.com",
+    "url" : "https://github.com/EdenTurgeman"
+  }],
```
Comment (AlonMiz) [inline package.json:30]: add me as well 🕵️‍♂️

**Review by AlonMiz — APPROVED** [review]
so we'll execute again only on explicit false

File: package.json (hunk 1 above)
Comment (EdenTurgeman) [reply package.json:30]: אתה author חחחחחח לא ocntributor

**Comment by EdenTurgeman** [comment]
> so we'll execute again only on explicit false

Or an exception

**Pull request merged**